src/styles/star.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
```

Comment:
You can't put external placement styles in the BEM block styles. If you need to place a BEM block inside another block, use mixes for that. 
Here is some more info https://en.bem.info/forum/4/